CHANGELOG.md

```diff
@@ -1460,6 +1460,10 @@ should use 4.0.1-alpha.0 for testing.
 
 -   Bug fix of `checkNetwork` in ENS (#5988)
 
+#### web3-eth-abi
+
+-   Bug fix of `ERR_UNSUPPORTED_DIR_IMPORT` in ABI (#6535)
+
```

Comment:
@kmlcnclk not here, under https://github.com/web3/web3.js/blob/4.x/packages/web3/CHANGELOG.md#unreleased and https://github.com/web3/web3.js/blob/4.x/packages/web3-eth-abi/CHANGELOG.md#unreleased , so I merge this PR. Thanks

Reply:
I am so sorry. I commit it now. Is it ok?